### PR TITLE
PXB-2261 : Adding PXB single platform test job

### PR DESCRIPTION
**Pull request by rameshvs02**

### CODE REVIEW

File: pxb/docker/test-binary-pxb24 [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+elif [[ ${XTRABACKUP_TARGET} == "galera56" ]]; then
+    wget --progress=dot:giga \
+      -O server-tarball/${XTRABACKUP_TARGET}/Percona-XtraDB-Cluster-$(uname -m).tar.gz \
+      https://www.percona.com/downloads/Percona-XtraDB-Cluster-56/Percona-XtraDB-Cluster-5.6.49-28.42-2/binary/tarball/Percona-XtraDB-Cluster-5.6.49-rel89.0-28.42.2.Linux.$(uname -m).ssl${SSL_VER}.tar.gz
+    rm -rf server
+    mkdir server
+    tar xfz server-tarball/${XTRABACKUP_TARGET}/Percona-XtraDB-Cluster-$(uname -m).tar.gz -C server
+    mv server/*/* server/
+else
+    sudo rm -rf bootstrap.sh
```
Comment (Sudokamikaze) [inline pxb/docker/test-binary-pxb24:65]: Please remove `rm -rf` of bootstrap and make it like: https://github.com/Percona-Lab/jenkins-pipelines/commit/2669588e2ea26a6756e8e4bb34a3998523b33198

**Comment by Sudokamikaze** [comment]
Overall looks good for me, have you tested it?

File: pxb/docker/pxb24-single-platform-run.groovy [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        skipDefaultCheckout()
+        skipStagesAfterUnstable()
+        timeout(time: 6, unit: 'DAYS')
+        buildDiscarder(logRotator(numToKeepStr: '200', artifactNumToKeepStr: '200'))
+    }
+    stages {
+        stage('Check out and Build PXB') {
+            parallel {
+                stage('Build PXB24') {
+                    agent { label 'docker' }
```
Comment (Sudokamikaze) [inline pxb/docker/pxb24-single-platform-run.groovy:62]: PXB Jenkins doesn't have such runner

**Comment by rameshvs02** [comment]
> Overall looks good for me, have you tested it?

Tested it on pxc.cd.percona.com and it is working fine
FYI https://pxc.cd.percona.com/job/pxb-24-test-run/

File: pxb/docker/test-binary-pxb24 (hunk 1 above)
Comment (rameshvs02) [reply pxb/docker/test-binary-pxb24]: done

**Comment by Sudokamikaze** [comment]
> > Overall looks good for me, have you tested it?
> 
> Tested it on pxc.cd.percona.com and it is working fine
> FYI https://pxc.cd.percona.com/job/pxb-24-test-run/

Tests LGTM

Could you please test 8.0 as well? Thanks in advance :)

File: pxb/docker/pxb24-single-platform-run.groovy [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                            sh '''
+                                sg docker -c "
+                                    if [ \$(docker ps -q | wc -l) -ne 0 ]; then
+                                        docker ps -q | xargs docker stop --time 1 || :
+                                    fi
+                                    ./pxb/docker/run-build-pxb24 ${DOCKER_OS}
+                                " 2>&1 | tee build.log
+
+                                if [[ -f \$(ls pxb/sources/pxb24/results/*.tar.gz | head -1) ]]; then
+                                    until aws s3 cp --no-progress --acl public-read pxb/sources/pxb24/results/*.tar.gz s3://pxc-build-cache/${BUILD_TAG}/pxb24.tar.gz; do
```
Comment (Sudokamikaze) [inline pxb/docker/pxb24-single-platform-run.groovy:90]: Should we use dedicated cache for PXB instead of PXC one?

Comment (rameshvs02) [reply pxb/docker/pxb24-single-platform-run.groovy]: yes, updated script.

**Comment by rameshvs02** [comment]
> > > Overall looks good for me, have you tested it?
> > 
> > 
> > Tested it on pxc.cd.percona.com and it is working fine
> > FYI https://pxc.cd.percona.com/job/pxb-24-test-run/
> 
> Tests LGTM
> 
> Could you please test 8.0 as well? Thanks in advance :)

PXB-8.0 job is working fine in PXC Jenkins host
https://pxc.cd.percona.com/job/pxb-80-test-run/

**Review by Sudokamikaze — APPROVED** [review]
LGTM, both code & test runs are good

**Comment by Sudokamikaze** [comment]
@hors PTAL
https://pxb.cd.percona.com/job/pxb80-single-platform-run/2/
https://pxb.cd.percona.com/job/pxb24-single-platform-run/2/
(ignore the access denied errs)

**Review by hors — APPROVED** [review]

**Pull request merged**